src/styles/main.scss

```diff
@@ -1,3 +1,36 @@
-body {
-  margin: 0;
+@import "./_variables/variables";
+@import "./_variables/reset";
+@import "./_variables/functions";
+
+.container {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  padding: $padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 7 * ($size + $gap) - $gap + 2 * $padding;
+
+  &__day {
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    width: $size;
+    height: $size;
+    background-color: $color;
+    list-style: none;
+    border: $border;
+    transition: all 0.5s ease;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
move `cursor: pointer;` to static state `&__day`

Reply:
done